Source/WebCore/rendering/GridBaselineAlignment.h

```diff
@@ -74,12 +74,12 @@ class GridBaselineAlignment {
     bool isOrthogonalChildForBaseline(const RenderBox&) const;
     bool isParallelToBaselineAxisForChild(const RenderBox&, GridAxis) const;
 
-    typedef HashMap<unsigned, std::unique_ptr<BaselineContext>, DefaultHash<unsigned>, WTF::UnsignedWithZeroKeyHashTraits<unsigned>> BaselineContextsMap;
+    typedef HashMap<unsigned, std::unique_ptr<BaselineAlignmentState>, DefaultHash<unsigned>, WTF::UnsignedWithZeroKeyHashTraits<unsigned>> BaselineAlignmentStatesMap;
 
     // Grid Container's WritingMode, used to determine grid item's orthogonality.
     WritingMode m_blockFlow;
-    BaselineContextsMap m_rowAxisAlignmentContext;
-    BaselineContextsMap m_colAxisAlignmentContext;
+    BaselineAlignmentStatesMap m_rowAxisAlignmentContext;
+    BaselineAlignmentStatesMap m_colAxisAlignmentContext;
```

Comment:
m_rowAxisAlignmentContext/m_colAxisAlignmentContext -> some other name?

Reply:
Changed to `m_row/columnAxisBaselineAlignmentContexts`

Also changed `BaselineAlignmentStatesMap` to `BaselineAlignmentStateMap` since the map part of the name already implies multiple states

Reply:
> Changed to `m_row/columnAxisBaselineAlignmentContexts`
> 
> Also changed `BaselineAlignmentStatesMap` to `BaselineAlignmentStateMap` since the map part of the name already implies multiple states

is "context" postfix has some special meaning here that prevents you from going with "state" instead?

Reply:
oh sorry I completely mistyped that last part! I ended up changing it to state